src/large-sum/large-sum-utils.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  calculateSumCarryOver: arr => (arr.length * 9).toString().length,
```

Comment:
Can you explain what this function is doing?

Reply:
In this solution (overall), I am breaking a large number into a separated number array so that I can add one digit of each large number at a time.  eg 1234 -> [1,2,3,4].  When I sum all numbers at the same digit, I want to know what the effect that this sum will have to the overall calculation.  Given that the largest single digit is 9 and that the amount of numbers being summed will be the length of the array, if the length of the array is 100, the largest possible result of summing all single digit numbers is 900.  In this case, the function will return "3" because the amount of digits in 900 is 3. With this number, I can know how many digits this part of the calculation will effect on the overall calculation.